crates/ruff_python_parser/tests/snapshots/invalid_syntax@implicitly_concatenated_unterminated_string.py.snap

```diff
@@ -160,6 +160,15 @@ Module(
   |
 
 
+  |
+1 | 'hello' 'world
+  |               ^ Syntax Error: Expected a statement
+2 | 1 + 1
+3 | 'hello' f'world {x}
+4 | 2 + 2
+  |
+
+
```

Comment:
I think this might get fixed if we start emitting `String` tokens instead of `Unknown` token because currently the parser thinks that the statement ended after `'hello'` but then it encountered an `Unknown` token (for unterminated string) and a `Newline` token both of which doesn't start or end a statement.